### PR TITLE
Transliterate Marathi to its own original Modi Script

### PR DESCRIPTION
**Pull request by ksinkar**
Transliterating Marathi to its original [Modi Script](https://en.wikipedia.org/wiki/Modi_script). After the British came to power in India after defeating the Marathas, they discontinued the use of the Modi script to repress national sentiments in a fashion [similar](https://www.economist.com/books-and-arts/2022/01/15/dominant-languages-can-spread-even-without-coercion) to the way they did [or still do](https://youtu.be/U0bgSbJUBG0) for [Welsh nationalism](http://www.welshnot.com/about/). 

Finally the[ Modi Script is now digitized](https://modi-script-working-group.gitlab.io/) and we can now have the Marathi Language Translation in its native script.

If the script is not rendered properly on your computer, please download the font from here: https://fonts.google.com/noto/specimen/Noto+Sans+Modi

### CODE REVIEW

**Comment by ksinkar** [comment]
@abhidesh128 Please review the transliteration for correction and compatibility with the Balbodh/Devanagari text.

File: content/version/1/4/code-of-conduct.mr.md [hunk 1]
```diff
@@ -3,76 +3,76 @@ version = "1.4"
...
-* इतर आचरण जे वाजवी हिशोबाने व्यावसायिक वातावरणात अयोग्य मानले जाऊ शकतात
+* 𑘩𑘺𑘽𑘐𑘲𑘎 𑘥𑘰𑘬𑘰 𑘎𑘲𑘽𑘪𑘰 𑘢𑘿𑘨𑘝𑘲𑘦𑘹𑘓𑘰 𑘪𑘰𑘢𑘨 𑘁𑘜𑘲 𑘀𑘡𑘲𑘬𑘿𑘘 𑘎𑘰𑘦𑘭𑘲𑘎 𑘩𑘎𑘿𑘬 𑘎𑘲𑘽𑘪𑘰 𑘮𑘩𑘿𑘩𑘰
+* 𑘪𑘲𑘪𑘰𑘟𑘭𑘿𑘢𑘟, 𑘀𑘢𑘦𑘰𑘡𑘕𑘡𑘎 / 𑘡𑘲𑘽𑘟𑘡𑘲𑘧 𑘘𑘲𑘢𑘿𑘢𑘜𑘿𑘧𑘰 𑘁𑘜𑘲 𑘪𑘺𑘧𑘎𑘿𑘝𑘲𑘎 𑘎𑘲𑘽𑘪𑘰 𑘎𑘳𑘘𑘡𑘺𑘝𑘲𑘎 𑘮𑘩𑘿𑘩𑘹
+* 𑘭𑘰𑘨𑘿𑘪𑘕𑘡𑘲𑘎 𑘎𑘲𑘽𑘪𑘰 𑘏𑘰𑘕𑘐𑘲 𑘔𑘯
+* 𑘭𑘿𑘢𑘬𑘿𑘘 𑘢𑘨𑘪𑘰𑘡𑘐𑘲𑘫𑘲𑘪𑘰𑘧 𑘃𑘝𑘨𑘰𑘽𑘓𑘲 𑘏𑘰𑘕𑘐𑘲 𑘦𑘰𑘮𑘲𑘝𑘲 𑘢𑘿𑘨𑘎𑘰𑘫𑘲𑘝 𑘎𑘨𑘜𑘹, 𑘄𑘟𑘰𑘮𑘨𑘜𑘰𑘝 𑘥𑘼𑘐𑘻𑘩𑘲𑘎
+  𑘎𑘲𑘽𑘪𑘰 𑘪𑘲𑘟𑘿𑘧𑘳𑘝𑘲𑘧 𑘢𑘝𑘿𑘝𑘰
+* 𑘃𑘝𑘨 𑘁𑘓𑘨𑘜 𑘕𑘹 𑘪𑘰𑘕𑘪𑘲 𑘮𑘲𑘫𑘻𑘤𑘰𑘡𑘹 𑘪𑘿𑘧𑘰𑘪𑘭𑘰𑘧𑘲𑘎 𑘪𑘰𑘝𑘰𑘪𑘨𑘜𑘰𑘝 𑘀𑘧𑘻𑘐𑘿𑘧 𑘦𑘰𑘡𑘩𑘹 𑘕𑘰𑘄 𑘫𑘎𑘝𑘰𑘝
 
-## आमच्या जबाबदार्या
+## 𑘁𑘦𑘓𑘿𑘧𑘰 𑘕𑘤𑘰𑘤𑘟𑘰𑘨𑘿𑘧𑘰
```
Comment (abhi-deshpande) [inline content/version/1/4/code-of-conduct.mr.md:34]: 𑘁𑘦𑘓𑘿𑘧𑘰 𑘕𑘤𑘰𑘤𑘟𑘰𑘨𑘿‍𑘧𑘰

File: content/version/1/4/code-of-conduct.mr.md [hunk 2]
```diff
@@ -3,76 +3,76 @@ version = "1.4"
...
+𑘦𑘻𑘎𑘯𑘹 𑘁𑘜𑘲 𑘭𑘿𑘪𑘰𑘐𑘝𑘰𑘨𑘿𑘮 𑘪𑘰𑘝𑘰𑘪𑘨𑘜 𑘑𑘚𑘪𑘲𑘜𑘿𑘧𑘰𑘓𑘰 𑘮𑘲𑘝𑘰𑘡𑘹 𑘁𑘦𑘿𑘮𑘲, 𑘧𑘻𑘐𑘟𑘰𑘝𑘰 𑘁𑘜𑘲 𑘢𑘰𑘩𑘡𑘎𑘨𑘿𑘝𑘰,
+𑘢𑘿𑘨𑘝𑘲𑘕𑘿𑘗𑘰 𑘑𑘹𑘝𑘻 𑘎𑘲 𑘁𑘦𑘓𑘿𑘧𑘰 𑘢𑘿𑘨𑘎𑘩𑘿𑘢𑘰𑘝 𑘁𑘜𑘲 𑘭𑘦𑘳𑘟𑘰𑘧𑘰𑘝 𑘭𑘮𑘥𑘰𑘐 𑘮𑘰 𑘭𑘐𑘯𑘿𑘧𑘰𑘽𑘭𑘰𑘙𑘲, 𑘪𑘧,
+𑘫𑘪𑘰𑘎𑘰𑘨, 𑘀𑘢𑘒𑘿𑘐𑘝𑘿𑘪, 𑘪𑘽𑘫, 𑘩𑘺𑘒𑘿𑘐𑘲𑘎 𑘪𑘺𑘫𑘲𑘬𑘿𑘘𑘿𑘧𑘹, 𑘩𑘺𑘒𑘿𑘐𑘲𑘎 𑘀𑘪𑘝𑘰𑘨, 𑘀𑘡𑘳𑘥𑘪, 𑘫𑘲𑘎𑘿𑘬𑘜, 𑘁𑘨𑘿𑘞𑘲𑘎
+𑘭𑘿𑘞𑘲𑘝𑘲, 𑘨𑘰𑘬𑘿𑘘𑘿𑘨𑘲𑘧𑘝𑘿𑘪, 𑘪𑘺𑘧𑘎𑘿𑘝𑘲𑘎 𑘀𑘪𑘝𑘰𑘨, 𑘕𑘰𑘝, 𑘢𑘡𑘿𑘞 𑘎𑘲𑘽𑘪𑘰 𑘎𑘰𑘦𑘭𑘲𑘎 𑘁𑘪𑘚, 𑘧𑘰𑘽𑘓𑘲 𑘢𑘨𑘿𑘪𑘰 𑘡
+𑘎𑘨𑘝𑘰, 𑘊𑘎 𑘔𑘯𑘦𑘳𑘎𑘿𑘝 𑘀𑘡𑘳𑘥𑘪 𑘨𑘰𑘮𑘲𑘩.
 
-## आमचे मापदण्ड
-सकारात्मक वातावरण तयार करण्यात योगदान देणार्या स्वभावाची उदाहरणे समाविष्टतात:
+## 𑘁𑘦𑘓𑘹 𑘦𑘰𑘢𑘟𑘜𑘿𑘚
+𑘭𑘎𑘰𑘨𑘰𑘝𑘿𑘦𑘎 𑘪𑘰𑘝𑘰𑘪𑘨𑘜 𑘝𑘧𑘰𑘨 𑘎𑘨𑘜𑘿𑘧𑘰𑘝 𑘧𑘻𑘐𑘟𑘰𑘡 𑘟𑘹𑘜𑘰𑘨𑘿𑘧𑘰 𑘭𑘿𑘪𑘥𑘰𑘪𑘰𑘓𑘲 𑘄𑘟𑘰𑘮𑘨𑘜𑘹 𑘭𑘦𑘰𑘪𑘲𑘬𑘿𑘘𑘝𑘰𑘝:
```
Comment (abhi-deshpande) [inline content/version/1/4/code-of-conduct.mr.md:17]: 𑘭𑘎𑘰𑘨𑘰𑘝𑘿𑘦𑘎 𑘪𑘰𑘝𑘰𑘪𑘨𑘜 𑘝𑘧𑘰𑘨 𑘎𑘨𑘜𑘿𑘧𑘰𑘝 𑘧𑘻𑘐𑘟𑘰𑘡 𑘟𑘹𑘜𑘰𑘨𑘿‍𑘧𑘰 𑘭𑘿𑘪𑘥𑘰𑘪𑘰𑘓𑘲 𑘄𑘟𑘰𑘮𑘨𑘜𑘹 𑘭𑘦𑘰𑘪𑘲𑘬𑘿𑘘𑘝𑘰𑘝:

File: content/version/1/4/code-of-conduct.mr.md [hunk 3]
```diff
@@ -3,76 +3,76 @@ version = "1.4"
...
 
-* स्वागतार्ह आणि सर्वसमावेशक भाषा वापरणे
-* भिन्न दृष्टीकोन आणि अनुभवांचा आदर करणे
-* कृतज्ञपणे विधायक टीका स्वीकारणे
-* समुदायाचा हिता कडे लक्ष केंद्रित करणे
-* समुदायाचा इतर सदस्यांबद्दल सहानुभूती दर्शविणे
+* 𑘭𑘿𑘪𑘰𑘐𑘝𑘰𑘨𑘿𑘮 𑘁𑘜𑘲 𑘭𑘨𑘿𑘪𑘭𑘦𑘰𑘪𑘹𑘫𑘎 𑘥𑘰𑘬𑘰 𑘪𑘰𑘢𑘨𑘜𑘹
+* 𑘥𑘲𑘡𑘿𑘡 𑘟𑘿𑘨𑘲𑘬𑘿𑘘𑘲𑘎𑘻𑘡 𑘁𑘜𑘲 𑘀𑘡𑘳𑘥𑘪𑘰𑘽𑘓𑘰 𑘁𑘟𑘨 𑘎𑘨𑘜𑘹
+* 𑘎𑘿𑘨𑘲𑘝𑘕𑘿𑘗𑘢𑘜𑘹 𑘪𑘲𑘠𑘰𑘧𑘎 𑘘𑘲𑘎𑘰 𑘭𑘿𑘪𑘲𑘎𑘰𑘨𑘜𑘹
+* 𑘭𑘦𑘳𑘟𑘰𑘧𑘰𑘓𑘰 𑘮𑘲𑘝𑘰 𑘎𑘚𑘹 𑘩𑘎𑘿𑘬 𑘎𑘹𑘽𑘟𑘿𑘨𑘲𑘝 𑘎𑘨𑘜𑘹
```
Comment (abhi-deshpande) [inline content/version/1/4/code-of-conduct.mr.md:22]: 𑘭𑘦𑘳𑘟𑘰𑘧𑘰𑘓𑘿𑘧𑘰 𑘮𑘲𑘝𑘰 𑘎𑘚𑘹 𑘩𑘎𑘿𑘬 𑘎𑘹𑘽𑘟𑘿𑘨𑘲𑘝 𑘎𑘨𑘜𑘹

**Comment by CoralineAda** [comment]
THanks @ksinkar ! I put out a call for native Marathi speakers to review the translation. https://twitter.com/EthicalOSS/status/1503445696820744192

**Comment by klaude** [comment]
> If the script is not rendered properly on your computer, please download the font from here: https://fonts.google.com/noto/specimen/Noto+Sans+Modi

The font rendered properly in macOS for me, but I ran into this issue when checking out the new translation from a Windows 10 machine. It looks like we can package the font as a web font, so it may be possible to render the page for all users without having to manually download a font. We'd have to see if hugo can handle a remote font download and apply it for just this page. An alternative is to add a note to the top or bottom of the translation to download the font if the page doesn't render properly. 

I don't think this should block merging, but it's something to keep in mind to fix up later.

**Review by CoralineAda — APPROVED** [review]
Thanks so much!

**Pull request merged**